README.md

```diff
@@ -48,8 +43,7 @@ $ ENC_PASSWORD=$(go run main.go auth encode-password <password> | cur -d' ' -f2)
 $ curl http://localhost:9410/api/v0/cs/<cs-id> -H 'content-type: application/json' -d '{"securityProfile":0,"base64SHA256Password":"'$ENC_PASSWORD'"}'
 ```
 
-Tokens must also be registered with the CSMS before they can be used. This can also be done using the
-[manager API](./manager/api/API.md). e.g.:
+Tokens are an authentication mechanism that needs to be set before use:
```

Comment:
This sounds odd. Tokens are not an authentication mechanism in the way that most people will think about it. Tokens are an EV concept.